src/node_lib.h

```diff
@@ -104,37 +159,74 @@ namespace node { namespace lib {
      * Convenience operations
      *********************************************************/
 
-    /*
-    Adds a new JavaScript module to the *running* Node.js engine.
-    */
+    /**
+     * @brief Adds a NPM module to the current JavaScript context.
+     * 
+     * Adds a given NPM module to the JavaScript context.
+     * This is achieved by calling `require('module_name')`.
+     * *Important* Make sure the NPM module is installed before using this method.

```

Comment:
Where does this module need to be installed? Globally? Or locally (which folder though?)?

Reply:
Here's how node finds its modules: https://nodejs.org/api/modules.html#modules_all_together
Note: We haven't properly solved this with our qt app (yet).